react/typings/vtex.css-handles.d.ts

```diff
@@ -0,0 +1,19 @@
+declare module 'vtex.css-handles' {
+  type CssHandlesInput = readonly string[]
+  type ValueOf<T extends readonly any[]> = T[number]
+
+  export type CssHandles<T extends CssHandlesInput> = Record<ValueOf<T>, string>
+
+  interface CssHandlesOptions {
+    migrationFrom?: string | string[]
+  }
+
+  export const useCssHandles: <T extends CssHandlesInput>(
+    handles: T,
+    options: CssHandlesOptions = {}
+  ) => CssHandles<T>
+  export const applyModifiers: (
+    handles: string,
+    modifier: string | string[]
+  ) => string
+}
```

Comment:
I don't think you need this anymore! The types that are installed by `vtex setup --typings` are correct

Reply:
I wasn't aware of this command. Thanks!